### ISSUE TITLE
Quiet mode coded to not work

### ISSUE THREAD

**Issue by vqmalic** [Breakthrough/PySceneDetect#56]
Thank you for this useful tool. It's really useful in the work I'm doing right now.

In manager.py, the SceneManager object takes quiet_mode as a Boolean keyword argument, but line 64 always initializes it to False.

**Comment by Breakthrough** [comment]
Good catch, thank you @vqmalic!

**Comment by Breakthrough** [comment]
The SceneManager class is going under a major overhaul just to give you a heads up, have started a complete rewrite of the project.  The new API is being developed following test driven development principles and will have plenty of unit tests to show example usage and error handling.

If you'd like a preview of some of the work or to give any feedback/suggestions you have for things you would like to see in the new API (or things in the old API that you think I might overlook during the transition) please let me know here #57

**Comment by Breakthrough** [comment]
In v0.5, in addition to the `--quiet/-q` flag, there will also be an `--info-level/-il` option which has values `none`, `debug`, `info` (default), and `error` which is the minimum level of messages to display.  Setting `-il none` in v0.5 will suppress all non-essential output, which is what the same thing `-q` flag used to do in v0.4 and previous.

In v0.5, the `-q` flag overrides the current info-level with `none`, achieving the same result (suppression of non-essential output).

I have completed the necessary changes for v0.5 and committed them to that version's development branch, and marked this issue as complete.  I will leave it open, however, until the changes have been merged with master, and the v0.5 release is out.
